index.html

```diff
@@ -36,193 +48,290 @@
+          <li>
+            <a href="https://www.linkedin.com/in/chiedozie013"
+              ><i class="fab fa-linkedin"></i
+            ></a>
+          </li>
+        </ul>
       </section>
-      <section class="works-section">
-        <h2>My Recent Works</h2>
-        <img src="./images/Img Placeholder.png" alt="workImg" />
-        <div class="workDetails">
-          <h3>Multi-Post Stories</h3>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard dummy text ever
-            since the 1500s, when an unknown printer took a standard dummy text.
-          </p>
-          <ul>
-            <li>CSS</li>
-            <li>HTML</li>
-            <li>Bootstrap</li>
-            <li>Ruby</li>
-          </ul>
-          <button class="btn" href="#">See project</button>
+      <section id="portfolio">
+        <div class="works-section">
+          <h2>My Recent Works</h2>
+          <div class="works-vector"></div>
         </div>
-      </section>
-      <section>
-        <div class="workCard">
-          <h2>Professional Art Printing Data</h2>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. Has been the industry's standard.
-          </p>
-          <ul>
-            <li>HTML</li>
-            <li>Bootstrap</li>
-            <li>Ruby</li>
-          </ul>
-          <button class="btn" href="#">See project</button>
+        <div class="works">
+          <img src="./images/Img Placeholder.png" alt="workImg" />
+          <div class="workDetails">
+            <h3>Multi-Post Stories</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. This has been the industry's standard dummy
+              text ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul class="work-lang">
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button class="btn" href="#">See project</button>
+          </div>
         </div>
-        <div class="workCard">
-          <h2>Professional Art Printing Data</h2>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. Has been the industry's standard.
-          </p>
-          <ul>
-            <li>HTML</li>
-            <li>Bootstrap</li>
-            <li>Ruby</li>
-          </ul>
-          <button class="btn" href="#">See project</button>
+      </section>
+      <section class="works-main">
+        <div class="workCard1">
+          <div class="position">
+            <div class="hide-card">
+              <h2 class="card-title">Professional Art Printing Data</h2>
+              <p class="card-details">
```

Comment:
- [ ] Just as the previous reviewer noted, I would recommend you change the styling to use grid instead of flex, by using grid, you will have access to grid-template-column and you will style it accordingly. You can read more on the usage [here](https://www.w3schools.com/cssref/pr_grid.php)

![image](https://github.com/chiedozie013/portfolio-microverse/assets/86200356/383abb05-1cda-425b-a214-5e04394e3046)